docs/learn/learn-teleport.md

```diff
@@ -14,25 +14,34 @@ and [SPREE modules][], which together ensure that assets are not lost or duplica
 
 ## How to Teleport
 
+Here are a couple of things that you need to understand before submitting a teleport transaction
+
+- There will be a transaction fee imposed on the source chain. This fee is **not** automatically deducted from the teleport amount you plan to send. Your account balance should be greater than the teleport amount and the transaction fees. If you do not account for the sending fee, the teleport transaction will fail.
+
+- There will be a transaction fee imposed at the destination chain. This fee is automatically deducted from the teleport amount you send. After this fee is deducted, the remaining account balance on the destination chain should be greater than its Existential Deposit. If not, the teleport transaction will fail.
```

Comment:
will fail (but you will have been charged in both locations?)

Reply:
I could test again and confirm. As far as I can recollect, It did not charge me the receiving fee